src/Microsoft.ML/Models/BinaryClassificationMetrics.cs

```diff
@@ -18,41 +19,53 @@ private BinaryClassificationMetrics()
         {
         }
 
-        internal static BinaryClassificationMetrics FromMetrics(IHostEnvironment env, IDataView overallMetrics, IDataView confusionMatrix)
+        internal static List<BinaryClassificationMetrics> FromMetrics(IHostEnvironment env, IDataView overallMetrics, IDataView confusionMatrix, int confusionMatriceStartIndex = 0)
         {
             Contracts.AssertValue(env);
             env.AssertValue(overallMetrics);
             env.AssertValue(confusionMatrix);
 
             var metricsEnumerable = overallMetrics.AsEnumerable<SerializationClass>(env, true, ignoreMissingColumns: true);
             var enumerator = metricsEnumerable.GetEnumerator();
+
             if (!enumerator.MoveNext())
             {
-                throw env.Except("The overall RegressionMetrics didn't have any rows.");
+                throw env.Except("The overall RegressionMetrics didn't have sufficient rows.");
```

Comment:
>sufficient [](start = 76, length = 10)

The old error message was more specific and accurate. You don't mention what *would* be sufficient. "Any" makes this clear.